modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -45,6 +45,16 @@ The latest schema is cached periodically. The cache period is defined by the clu
 
 Default for `redpanda.iceberg.mode`. Disables writing to an Iceberg table for the topic.
 
+[NOTE]
+====
+The following modes are compatible with producing to an Iceberg topic using Redpanda Console:
```

Comment:
@rockwotj mind signing off on this? Thank you

Reply:
Yes this information is correct

Reply:
@rockwotj did we confirm its impossible to get Console to product using the wire format (whereas rpk always does!) cc @r-vasquez

Reply:
It would be cool to support json or textproto for console testing (with or without wire format)